movieApp.js

```diff
@@ -0,0 +1,423 @@
+const movies = [
+  {
+    id: 1,
+    title: "Interstellar",
+    description:
```

Comment:
This is a lot of data. For next time I would recommend looking into having this entire array in a separate file to make this main JS logic file smaller.

Reply:
Thanks for suggestion, I implemented it